config.py

```diff
@@ -137,14 +147,17 @@ class Live(Config):
 
 class Preview(Live):
     FEATURE_FLAGS_NEW_SUPPLIER_FLOW = enabled_since('2017-02-06')
+    GOOGLE_TAG_MANAGER = dict(Config.GOOGLE_TAG_MANAGER, environmentID='env-7', authToken='Kpjle7g00efQOB2j1o3fVQ')
```

Comment:
They are generated auto-inc keys in GTM, as far as I can see. Some environments are created automatically: "Live" and "Latest" (which I am taking to mean "Dev") are there from the start. Then I added Staging and Preview as two more custom ones. I might have created and deleted one or two more in between :) So yeah it's a bit random.